lib/single-new-eks-opensource-observability-pattern/index.ts

```diff
@@ -52,9 +65,16 @@ export default class SingleNewEksOpenSourceobservabilityPattern {
             );
         }
 
+        if (utils.valueFromContext(scope, "apiserver.pattern.enabled", false)) {
+            ampAddOnProps.enableAPIServerJob = true,
+            ampAddOnProps.ampRules?.ruleFilePaths.push(
+                __dirname + '/../common/resources/amp-config/apiserver/recording-rules.yml'
+            );
+        }
+
         if (utils.valueFromContext(scope, "nginx.pattern.enabled", false)) {
             ampAddOnProps.openTelemetryCollector = {
-                manifestPath: __dirname + '/../common/resources/otel-collector-config.yml',
+                manifestPath: __dirname + '/../common/resources/otel-collector-config-new.yml',
```

Comment:
Are you writing this to an internal new file which will be used only while deployment?

Reply:
Yes writing to a temporary file.